frameworks/CSharp/aspnetcore/config.toml

```
@@ -262,6 +277,19 @@ platform = ".NET"
 webserver = "Kestrel"
 versus = "aspcore"
 
+[aot]
+urls.plaintext = "/plaintext"
+urls.json = "/json"
+approach = "Realistic"
```

NativeAOT is not currently supported for ASP.NET Core apps and thus this should not be marked as "Realistic".

```suggestion
approach = "Stripped"
```

---

> NativeAOT is not currently supported for ASP.NET Core apps and thus this should not be marked as "Realistic".

I didn't realize we support mono with asp.net:

https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/CSharp/aspnetcore-mono/config.toml#L7

If this is to avoid confusion, it doesn't look like a realistic marking leads to any. Mono has been marked realistic for years.